pages/pil/read/views/index.jsx

```diff
@@ -19,7 +19,8 @@ const PIL = ({
   canUpdate,
   allowedActions,
   openTask,
-  correctEstablishment
+  correctEstablishment,
```

Comment:
👀 

As opposed to `incorrectEstablishment`?

My guess is that the establishment that holds the PIL appears on the pdf version? Not necessarily the establishment that is requesting the document?

Reply:
Not my code, I just added `currentPath`.

Reply:
Yeah, it just made me aware of the issue with potentially leaking establishment details.